lib/rubocop/cop/performance/string_replacement.rb

```diff
@@ -164,13 +165,22 @@ def require_double_quotes?(string)
             StringHelp::ESCAPED_CHAR_REGEXP =~ string
         end
 
+        def require_closing_paren?(node)
+          source = node.loc.expression.source
```

Comment:
That's definitely a bad idea. Doing string checks in the entire source of an expression is a recipe for bugs.